app/components/landscape-portrait-slider.js

```diff
@@ -1,32 +1,242 @@
 // https://github.com/EnCiv/undebate-ssp/issues/120
 
-import React, { useState, useEffect, useRef } from 'react'
+import React, { useState } from 'react'
 import { createUseStyles } from 'react-jss'
 import cx from 'classnames'
+import SvgDesktop from '../svgr/desktop'
+import SvgPortrait from '../svgr/portrait'
+import SvgRectangle from '../svgr/rectangle'
+import SvgExternalLink from '../svgr/external-link'
 
+const daLink = (
+    <a
+        style={{
+            textDecorationLine: 'none',
+            color: 'grey',
+            fontFamily: 'poppins',
+        }}
+        href={'https://cc.enciv.org/country:us/organization:ucla-student-accociation/office:usac-president/2021-05-07'}
+    >
+        District Attorney Election
```

Comment:
The values for this need to come from the linkObj property passed to the LandscapePortraitSlider component.  (not hard coded).

Also, in the figma, the purple line goes up the left of all the text, not just the last line.   You can probably wrap all this in a div and give it a border-left.